### DISCUSSION TITLE
cura 4.20.....can't print hollow pipe,

### DISCUSSION THREAD

**Discussion by raspsigi** [smartavionics/Cura#233] [General]
Hi , I can't slice hollow pipes  or even  a test calibration cube, what can be the cause

**Comment by smartavionics** [comment]
Please provide a project file so I can see the settings you are using. Save project and then either zip up the .3mf file or rename the .3mf file to .zip and attach here. Thanks.

**Reply by raspsigi** [reply]
[CFFFP_Rohrbogen.zip](https://github.com/smartavionics/Cura/files/12546773/CFFFP_Rohrbogen.zip)
it's the renamed .3mf
I changed back to the "Master" from March 22, slicing without any problem
greetings

**Reply by raspsigi** [reply]
btw I'm using Raspberry 4 8GB with Debian Bullseye

**Comment by smartavionics** [comment]
Hi, thanks for the file. I have sliced that OK on both 32 bit and 64 Debian buster systems. I don't have anything running bullseye.
If you run Cura from a terminal do you get any diagnostics printed? Or maybe you get something in ~/.local/share/cura/stdout.log or ~/.local/share/cura/stderr.log or ~/.local/share/cura/4.20/cura.log ? If so, please let me see that.

![Screenshot_2023-09-07_21-36-30](https://github.com/smartavionics/Cura/assets/585618/d8a0218c-861b-4e4f-999f-327f8e64657d)

**Reply by raspsigi** [reply]
check it, if it's hollow, outside my one is also perfect but, not hollow, but as I say with the "MASTER version" everything is perfect.
I forgot to say, I'm using the AppImage version
![Cura_Master](https://github.com/smartavionics/Cura/assets/144229293/28c43113-38a2-447a-9c66-3e2596629139)
![Cura_20 14](https://github.com/smartavionics/Cura/assets/144229293/f50c1dc1-d46b-4783-8054-60ac6809973c)

**Comment by smartavionics** [comment]
Sorry, I don't understand the problem. I see objects with infill. Are you saying that the objects should be hollow? Please explain more.

**Reply by raspsigi** [reply]
the holes you see in the big  picture are in the cube , one of them are half filled "it should be a empty 
 the other one are not a beauty but the hole are ok,
the big  thing should be  a pipe  not a filled body



![after_slicing](https://github.com/smartavionics/Cura/assets/144229293/643505f0-5819-401a-bdca-de1029b7f698)
![before_slicing](https://github.com/smartavionics/Cura/assets/144229293/0e4281b3-9294-4dd8-819d-5af5f275deba)

![cube_sliced](https://github.com/smartavionics/Cura/assets/144229293/6b3b8466-fd21-4ade-9b62-643989928ee5)

![cube_not_sliced](https://github.com/smartavionics/Cura/assets/144229293/f5da74eb-d6eb-4379-a835-fbf29b178714)
as  I say, it's only with version > 20....

**Comment by smartavionics** [comment]
Thanks for the info. I believe the problem is caused by the Make Overhang Printable setting being enabled. If you disable that, the pipe is hollow.

![Screenshot_2023-09-09_12-52-55](https://github.com/smartavionics/Cura/assets/585618/6f83239f-50f1-421d-b0fa-cfc75c5b047f)

**Comment by smartavionics** [comment]
The default value of that setting (not checked) has not been changed in many years so somehow you have enabled it or used a project that enabled it?

**Reply by raspsigi** [reply]
damned sooorrrrry you are right, I turned it on by accident. I don't how I did,  I'm using your AppImage since a long time and never had any problem
![grafik](https://github.com/smartavionics/Cura/assets/144229293/f03ffd4b-aef8-4a2e-ac76-d1ab13b52cd0)

**Comment by smartavionics** [comment]
No problem, it's easy to miss changed settings. Happy printing!
